### ISSUE TITLE
Unhandled Division By Zero exception

### ISSUE THREAD

**Issue by c0d3rk1d** [hirosystems/clarinet#525]
**Steps to reproduce the issue**

Open up the terminal and type
```
clarinet console
```

After REPL starts
```
note: no manifest found. Continuing with default.
clarity-repl v0.33.0
Enter "::help" for usage hints.
Connected to a transient in-memory database.
```

Try to divide any number to zero
```
>> (/ 0 0)
```

Console crashes and returns to terminal.
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Runtime(DivisionByZero, Some([]))', components/clarity-repl/src/repl/interpreter.rs:516:89
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by c0d3rk1d** [comment]
Similar crash happens when we get "ArithmeticUnderflow" error too...

**Comment by c0d3rk1d** [comment]
```
(- u0 u1)
```

**Comment by obycode** [comment]
Yup, thanks. Basically the problem is that the console is not catching those top-level errors. Should be an easy fix.

**Comment by lgalabru** [comment]
Addressed with #535
